libkbfs/data_types.go

```diff
@@ -195,6 +226,10 @@ const (
 	// FilesWithHolesDataVer is the data version for files
 	// with holes.
 	FilesWithHolesDataVer DataVer = 2
+	// BigFilesDataVer is the data version for files that have
```

Comment:
Good point.  I'm going with with `ChildHolesDataVer` and `AtLeastTwoLevelsOfChildrenDataVer`, leaving out the term "block" altogether since it's implied by "DataVer".  And I'm leaving the suffix as a namespace identifier to avoid `libkbfs` naming conflicts in the future.